integration-cli/docker_cli_network_unix_test.go

```diff
@@ -289,7 +289,8 @@ func (s *DockerNetworkSuite) TestDockerNetworkLsDefault(c *check.C) {
 }
 
 func (s *DockerSuite) TestNetworkLsFormat(c *check.C) {
-	testRequires(c, DaemonIsLinux)
+	// E2E: Test assumes only default networks.
+	testRequires(c, DaemonIsLinux, NotE2E)
```

Comment:
This test is only testing CLI behaviour, please delete it instead of skipping it (see #32866).

We already test the formatter in `docker/cli`